test/operators/dedicatedadmin.go

```diff
@@ -57,34 +57,31 @@ var roleBindings = []string{
 
 var _ = ginkgo.Describe("[OSD] Dedicated Admin Operator", func() {
 	h := helper.New()
-
-	// Check that the operator deployment exists in the operator namespace
-	ginkgo.Context("deployments", func() {
+	// Check that the operator configmap has been deployed
+	ginkgo.Context("configmaps", func() {
 		ginkgo.It("should exist", func() {
 			// Wait for lockfile to signal operator is active
+			// If this fails, all other tests will fail.
 			err := pollLockFile(h)
 			Expect(err).ToNot(HaveOccurred(), "failed fetching the configMap lockfile")
+		}, float64(globalPollingTimeout * 60)
+	})
 
+	// Check that the operator deployment exists in the operator namespace
+	ginkgo.Context("deployments", func() {
+		ginkgo.It("should exist", func() {
 			deployments, err := pollDeploymentList(h)
 
 			Expect(err).ToNot(HaveOccurred(), "failed fetching deployments")
 			Expect(deployments).NotTo(BeNil())
-		})
+		}, float64(globalPollingTimeout * 60)
```

Comment:
Kind of confused about the * 60s in this code. Rather than do this, why not make the globalPollingTimeout 30 * 60 and skip all of the multiplies?

Reply:
I would have defaulted to keeping the variable a human-friendly (heh, or just, non-math-y) single value, but you have a point too.  I could go either way and don't feel strongly about either.

Reply:
I'd elect for baking the 60 into the variable if you don't mind. I think it'll minimize complexity for adding in tests later.